VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -8,6 +8,19 @@
 
 import Foundation
 
+private typealias ResponseDictionaryClosure = (responseDictionary: VimeoClient.ResponseDictionary?) -> Void
+
+private protocol Cache
```

Comment:
Could this just be generic? This would mean that we don't have to specify the return type here, and mean that the method names wouldn't have to state the type.